params/json.libevm_test.go

```diff
@@ -144,3 +145,60 @@ func TestChainConfigJSONRoundTrip(t *testing.T) {
 		})
 	}
 }
+
+func Test_UnmarshalChainConfig(t *testing.T) {
```

Comment:
Please only include `_` in identifiers when it's required by a general convention; the only instance I can think of off the top of my head is the [`godoc` parsing of testable examples](https://go.dev/blog/examples#example-function-names).

Reply:
The `_` right after `Test` can be removed indeed. However, if we test for example `func (o *Object) myFunc()` naming the test as `TestObject_myFunc` is better than `TestObjectmyFunc` readability-wise, would you agree or have another suggestion? 🤔